generate/templates/manual/src/convenient_patch.cc

```diff
@@ -72,17 +76,19 @@ PatchData *createFromRaw(git_patch *raw) {
     bool EOFFlag = false;
     for (unsigned int j = 0; j < hunkData->numLines; ++j) {
       git_diff_line *storeLine = (git_diff_line *)malloc(sizeof(git_diff_line));
-      const git_diff_line *line;
-      git_patch_get_line_in_hunk(&line, raw, i, j);
+      const git_diff_line *line = NULL;
+      int result = git_patch_get_line_in_hunk(&line, raw, i, j);
+      if (result != 0) {
+        continue;
+      }
 
       if (j == 0) {
-        // calculate strlen only once for the first line of the first hunk.
```

Comment:
I guess my confusion is that (1) it's not like we were re-using the length, and (2) we're doing this for the first line of every hunk, not just the first line of the first hunk as the comment says.